src/main/java/uk/ac/cam/cl/dtg/util/mappers/UnimplementedMappingException.java

```diff
@@ -0,0 +1,16 @@
+package uk.ac.cam.cl.dtg.util.mappers;
+
+public class UnimplementedMappingException extends RuntimeException {
```

Comment:
What about extending `UnsupportedOperationException` instead? The end result is the same, but I think this exception is more meaningful and appropriate in this scenario.